reconcile/openshift_base.py

```diff
@@ -109,14 +105,10 @@ def init_specs_to_fetch(ri, oc_map,
         for cluster_info in clusters or []:
             cluster = cluster_info['name']
             oc = oc_map.get(cluster)
-            if oc is None:
-                msg = f"[{cluster}] cluster skipped."
-                logging.debug(msg)
-                continue
-            if oc is False:
-                ri.register_error()
-                msg = f"[{cluster}] cluster has no automationToken."
-                logging.error(msg)
+            if not oc:
+                if oc.log_level == 40:
```

Comment:
what about CRITICAL? FATAL? Basically anything from ERROR up? THey are even worse and should also make the integration fail.

Reply:
So is it better to use integer if I am checking for ERROR and higher
`if oc.log_level >= 40`
or preferable to just do
`if oc.log_level == 'ERROR' or oc.log_level == 'CRITICAL' or oc.log_level == 'FATAL':`

Reply:
I went with option 2 from above

Reply:
Neither. I mean, create the instance like this:
```python
OCLogMsg(log_level=logging.ERROR, message=...)
```
and check like this:
```python
if oc.log_level >= logging.ERROR:
    ...
```

Reply:
`FATAL` is not a logging constant
https://docs.python.org/3/library/logging.html#levels

Reply:
It is: https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L92

Reply:
:+1: